### DISCUSSION TITLE
List view filters for Relationships

### DISCUSSION THREAD

**Discussion by DiesIrae** [keystonejs/keystone#5574] [Feature requests]
The relationship fields are absent from the filter menu. 
When selection a relationship field, we should be able to choose between options like "contains", "does not contain", "exact"... with a multi-select field on the ref entity.

**Comment by bladey** [comment]
Thanks @DiesIrae, relationship filtering will be coming soon.

**Reply by DiesIrae** [reply]
Hi Thomas, very glad to read it! ☺️
